File: Phonebook.Contracts/Models/PhoneBookEntries/PhoneBook.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Phonebook.Contracts.Models.PhoneBookEntries
{
    public class Phonebook
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int EntriesCount => PhoneBookEntries.Count;
        public virtual IList<PhonebookEntry> PhoneBookEntries { get; set; } = new List<PhonebookEntry>();

    }
}

File: Phonebook.web/ClientApp/src/app/models/phonebook.ts
export interface IPhonebook {
  id: number;
  name: string;
  entriesCount: number;
}

export class Phonebook {
  Id: number;
  Name: string;
  EntriesCount: number;

  constructor() {
    this.Id = 0;
    this.Name = '';
    this.EntriesCount = 0;

  }
}
